scripts/create_backup_osgp_dbs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+HOST=localhost
+PORT=5432
+USER=osp_admin
+
+HOSTNAME=`hostname`
+TIMESTAMP=`date +%Y%m%d_%H%M%S`
+
+BACKUP_DIR=$HOME/backups/osgp/${HOSTNAME}_$TIMESTAMP
```

Comment:
Is it the intention to use the literal string 'hostname' or to use the environment variable `HOSTNAME` which expands to the name of the server/host?